src/main/java/io/fabric8/vertx/maven/plugin/mojos/SetupMojo.java

```diff
@@ -117,8 +117,8 @@ public void execute() throws MojoExecutionException, MojoFailureException {
                     }
                 }
 
-                if (verticle != null  && verticle.endsWith(".java")) {
-                    verticle = verticle.substring(0, verticle.length() -".java".length());
+                if (verticle != null && verticle.endsWith(".java")) {
+                    verticle = verticle.substring(0, verticle.length() - ".java".length());
```

Comment:
wondering if we need this, as a developer i know that parameter like this will always be a FQN .. its good to do this check for safety